packages/veritone-react-common/.storybook/config.js

```diff
@@ -1,14 +1,22 @@
-import React from 'react';
+import React, {Fragment} from 'react';
 import { configure, addDecorator } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
 
 import '../src/styles/global.scss';
+import { VSDKStyleWrapper } from '../src/helpers/withMuiThemeProvider';
 
 const req = require.context('../src', true, /story.js$/);
 
 function loadStories() {
   req.keys().forEach(filename => req(filename));
 }
 
+const withStyles = (story) => (
+  <VSDKStyleWrapper>
+    { story() }
+  </VSDKStyleWrapper>
+)
```

Comment:
this would have a semi if prettier was run, please make sure to run it over the whole feature